proxy/mitmproxy.py

```diff
@@ -37,8 +37,8 @@ def start(self):
 
         return self.process
 
-    def stop(self):
-        if self.mode is "record":
+    def stop(self, kill=True):
```

Comment:
Why is the optional kwarg added here? I don't see where we pass anything to this method.

Reply:
removed the argument and updated the stop function